platforms/lambda/README.md

```diff
@@ -1,121 +1,7 @@
-# Simple TypeScript Script Example
+# AWS Lambda
 
-This example shows how to use [Prisma Client](https://github.com/prisma/prisma-client-js) in a **simple TypeScript script** to read and write data in a database.
+AWS Lambda. Zips the file and uploads it using the `--zip-file` aws cli flag.
 
-## How to use
+# Friction points
```

Comment:
```suggestion
## Friction points
```

Reply:
Thanks, I replaced all occurrences of where # is used